blocks/address/src/app.tsx

```diff
@@ -0,0 +1,677 @@
+                );
+              }}
+              PaperComponent={({ children, ...props }) => (
+                <Paper
+                  {...props}
+                  sx={{
+                    filter:
+                      "drop-shadow(0px 11px 30px rgba(61, 78, 133, 0.04)) drop-shadow(0px 7.12963px 18.37px rgba(61, 78, 133, 0.05)) drop-shadow(0px 4.23704px 8.1px rgba(61, 78, 133, 0.06)) drop-shadow(0px 0.203704px 0.62963px rgba(61, 78, 133, 0.07))",
+                    border: ({ palette }) => `1px solid ${palette.gray[20]}`,
+                    boxShadow: "none",
+                    [`.${autocompleteClasses.listbox}`]: {
+                      padding: "0px",
+                      maxHeight: "unset",
+                      [`.${autocompleteClasses.option}`]: {
+                        alignItems: "flex-start",
+                        paddingX: ({ spacing }) => `${spacing(2.5)} !important`,
+                        paddingY: 1.25,
+                      },
+                    },
+                  }}
+                >
+                  {children}
+                </Paper>
+              )}
+              sx={{
+                [`.${autocompleteClasses.input}`]: {
+                  paddingLeft: "0 !important",
+                },
+                [`.${autocompleteClasses.inputRoot}`]: {
+                  paddingX: ({ spacing }) => `${spacing(2.75)} !important`,
+                },
+              }}
+            />
+
+            {suggestionsError ? (
+              <Box
+                display="flex"
+                flexDirection="column"
+                justifyContent="center"
+                gap={1}
+              >
+                <Typography
+                  variant="smallTextLabels"
+                  sx={{
+                    fontWeight: 700,
+                    fontSize: 13,
+                    lineHeight: 1,
+                    letterSpacing: "-0.02em",
+                    color: ({ palette }) => palette.black,
+                  }}
+                >
+                  <Box
+                    component="span"
+                    sx={{ color: ({ palette }) => palette.red[60] }}
+                  >
+                    Error connecting
+                  </Box>{" "}
+                  to the Block Protocol
+                </Typography>
+                <Typography
+                  sx={{
+                    fontWeight: 500,
+                    fontSize: 15,
+                    lineHeight: 1,
+                    letterSpacing: "-0.02em",
+                    color: ({ palette }) => palette.gray[50],
+                  }}
+                >
+                  Check your network connection or{" "}
+                  <Box
+                    component="span"
+                    sx={{
+                      fontWeight: 700,
+                      color: ({ palette }) => palette.gray[70],
+                    }}
+                  >
+                    contact support
+                  </Box>{" "}
```

Comment:
Since this is not a link, I would not bold it – it looks like it should be clickable

Reply:
I'm not sure if it's supposed to be a link. Here's the [design](https://www.figma.com/file/T10d2P2iNb1LdRC5a9J7vA/Blocks?node-id=1403%3A5185&t=qzUaNLUATCz9wAjo-4).

@nonparibus

Reply:
It was originally intended to but we don't have a way of EAs telling blocks what 'contact support' should do at the moment. We can make it a link later, but while it isn't I would not bold it as it implies interactivity.

Reply:
Addressed in 1f10fc11ee423380e7e0900acbec6876523ddb76